README.md

```diff
@@ -1,602 +1,387 @@
+| [`match_array`][]        |                   | all                             | for comparing arrays of ActiveRecord objects             |
+| [`have_been_enqueued`][] |                   | all                             | requires config: `ActiveJob::Base.queue_adapter = :test` |
+| [`have_enqueued_job`][]  |                   | all                             | requires config: `ActiveJob::Base.queue_adapter = :test` |
 
-To provide a layout for the render, you'll need to specify _both_ the template
-and the layout explicitly. For example:
+Follow the links above for examples of how each matcher is used.
 
-```ruby
-render :template => "events/show", :layout => "layouts/application"
-```
+[the matchers that come standard in RSpec]: https://relishapp.com/rspec/rspec-expectations/docs/built-in-matchers
+[`be_a_new`]: https://relishapp.com/rspec/rspec-rails/docs/matchers/be-a-new-matcher
+[`render_template`]: https://relishapp.com/rspec/rspec-rails/docs/matchers/render-template-matcher
+[`redirect_to`]: https://relishapp.com/rspec/rspec-rails/docs/matchers/redirect-to-matcher
+[`route_to`]: https://relishapp.com/rspec/rspec-rails/docs/routing-specs/route-to-matcher
+[`be_routable`]: https://relishapp.com/rspec/rspec-rails/docs/routing-specs/be-routable-matcher
+[`have_http_status`]: https://relishapp.com/rspec/rspec-rails/docs/matchers/have-http-status-matcher
+[`match_array`]: https://relishapp.com/rspec/rspec-rails/docs/matchers/activerecord-relation-match-array
+[`have_been_enqueued`]: https://relishapp.com/rspec/rspec-rails/docs/matchers/have-been-enqueued-matcher
+[`have_enqueued_job`]: https://relishapp.com/rspec/rspec-rails/docs/matchers/have-enqueued-job-matcher
 
-### `assign(key, val)`
+### What else does RSpec Rails add?
 
-Use this to assign values to instance variables in the view:
+For a comprehensive look at RSpec Rails’ features,
+read the [official Cucumber documentation][].
 
-```ruby
-assign(:widget, Widget.new)
-render
-```
+[official Cucumber documentation]: https://relishapp.com/rspec/rspec-rails/docs
 
-The code above assigns `Widget.new` to the `@widget` variable in the view, and
-then renders the view.
+## What tests should I write?
 
-Note that because view specs mix in `ActionView::TestCase` behavior, any
-instance variables you set will be transparently propagated into your views
-(similar to how instance variables you set in controller actions are made
-available in views). For example:
+At the risk of grossly oversimplifying,
+there are two kinds of tests you can write:
+those that verify
+**an individual component of your application** (unit tests),
+and those that verify
+**how multiple parts of your application work as a whole** (integration tests).
 
-```ruby
-@widget = Widget.new
-render # @widget is available inside the view
-```
+RSpec Rails defines ten different _types_ of specs (7 unit, 3 integration)
```

Comment:
Remove `(7 unit, 3 integration)` its confusing because all Rails style specs are integration specs formally, and only a couple are end to end.

Reply:
Done.